polaris-react/src/components/CustomProperties/CustomProperties.tsx

```diff
@@ -1,8 +1,6 @@
 import React from 'react';
-
-import type {ColorScheme} from '../../tokens';
-
-import {styles} from './styles';
```

Comment:
Want to flag this as I'm sure if a direct input of the css file will have the effect we are looking for.

Reply:
Fixed in this commit https://github.com/Shopify/polaris/pull/5659/commits/955da8e034fcc2b8e9e80077b56fcf0a153b3d40